src/OpenTelemetry/Logs/OpenTelemetryLoggerProvider.cs

```diff
@@ -48,7 +50,10 @@ internal OpenTelemetryLoggerProvider(OpenTelemetryLoggerOptions options)
         {
             Guard.ThrowIfNull(options);
 
-            this.Options = options;
+            this.IncludeScopes = options.IncludeScopes;
```

Comment:
Did you add a test targeting this change? I see a lot of test change, not sure if it was modifying existing ones alone, or there is on dedicated to testing the bug fix in this PR.

Reply:
No, those changes were just to fix the tests that this change broke.

I'll add new tests to verify my change.